tools/SetupFlow/DevHome.SetupFlow/ViewModels/AddRepoViewModel.cs

```diff
@@ -579,6 +579,8 @@ private void SelectionOptionsChanged(string selectedItem)
     [RelayCommand]
     private async Task AddAccountClicked()
     {
+        _addRepoDialog.Focus(FocusState.Programmatic);

```

Comment:
Can this be set on `Load` of `AddRepoDialog`? Since this is more of a UI behavior, XAML and/or code behind is a better candidate than the view model.

Reply:
> Can this be set on `Load` of `AddRepoDialog`? Since this is more of a UI behavior, XAML and/or code behind is a better candidate than the view model.

It can not because this isn't setting focus on load.  This is setting on a button click.  Furthermore, the MenuItems are made during runtime inside the ViewModel and refer to relay commands in the ViewModel.  [ConstructFlyout](https://github.com/microsoft/devhome/blob/main/tools/SetupFlow/DevHome.SetupFlow/ViewModels/AddRepoViewModel.cs#L736) is the method.  

I took some time to separate out the view and view model logic. The work needed is getting longer than the PR justifies.

Reply:
Closing because the isn't part of the load event and should not be put into the load event.  This event is when the Add Account button is clicked in the MenuFlyout of the Repo page.